### DISCUSSION TITLE
VcPrint 属性downloadAutomatically如何自定义文件名

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#172] [Q&A] [answered]
设置 downloadAutomatically属性后点击打印按钮，默认下载文件名无后缀，请问各位大神如何修改

**Comment by zouyaoji** [comment]
这个应该是bug，下个版本解决。

**Comment by zouyaoji** [comment, marked as answer]
已在 v3.0.12 中修复。默认取的 多语言配置里面的名称，如果有需要可以通过 VcPrint 的 screenshotName 属性传参。
